imagevector/charts.go

```diff
@@ -0,0 +1,12 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// Code generated by generate-imagename-constants.sh. DO NOT EDIT.
+
+package imagevector
+
+const (
+	// ChartImageNameGardenlet is a constant for an image in the image vector with name 'gardenlet'.
```

Comment:
constant for a chart, right?

Reply:
The packaged chart is also considered as "image", hence, I didn't bother with this unspecific doc string. I guess that's good enough, no?